test/helpers/ssh_command.go

```diff
@@ -269,10 +265,6 @@ func (client *SSHClient) RunCommandContext(ctx context.Context, cmd *SSHCommand)
 			if err != nil {
 				log.Errorf("write ^C error: %s", err)
 			}
-			err = session.Wait()
-			if err != nil {
-				log.Errorf("wait error: %s", err)
-			}
```

Comment:
I thought the whole point of these waits was to allow non-failfast to work? if we just exit this we don't know if any state is left sitting around, no? Granted, we often wrap ssh commands in WithTimeout or WithContext but we aren't consistent... and what happens if the cleanup function times out?